crates/op-rbuilder/src/metrics.rs

```diff
@@ -82,10 +82,14 @@ pub struct OpRBuilderMetrics {
     pub flashblock_time_drift: Histogram,
     /// Time offset we used for first flashblock
     pub first_flashblock_time_offset: Histogram,
+    /// Number of requests sent to the eth_sendBundle endpoint
+    pub bundle_requests: Counter,
     /// Number of valid bundles received at the eth_sendBundle endpoint
     pub bundles_received: Counter,
     /// Number of reverted bundles
     pub bundles_reverted: Histogram,
+    /// Time taken to respond to a request to the eth_sendBundle endpoint
+    pub bundle_receive_duration: Histogram,
```

Comment:
I believe the histogram already includes a counter so the bundles_received metric is redundant